data/chameleoncloud/sites/nu/clusters/chameleon/nodes/3d8645f2-2a9e-4012-aa50-3a5977c12289.json

```diff
@@ -0,0 +1,106 @@
+{
+  "architecture": {
+    "platform_type": "x86_64",
+    "smp_size": 2,
+    "smt_size": 48
+  },
+  "bios": {
+    "release_date": "12/04/2018",
+    "vendor": "Dell Inc.",
+    "version": "2.9.1"
+  },
+  "chassis": {
+    "manufacturer": "Dell Inc.",
+    "name": null,
+    "serial": null
+  },
+  "gpu": {
+    "gpu": false
+  },
+  "main_memory": {
+    "humanized_ram_size": "128 GiB",
```

Comment:
This was hard-coded in the generator--is this right?

Reply:
Yeah, I'm not sure why that is. Seems pretty bad